src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/trust/CorporateDto.java

```diff
@@ -0,0 +1,354 @@
+    private String roAddressCountry;
+
+    @JsonProperty("ro_address_locality")
+    private String roAddressLocality;
+
+    @JsonProperty("ro_address_po_box")
+    private String roAddressPoBox;
+
+    @JsonProperty("ro_address_postal_code")
+    private String roAddressPostalCode;
+
+    @JsonProperty("ro_address_premises")
+    private String roAddressPremises;
+
+    @JsonProperty("ro_address_region")
+    private String roAddressRegion;
+
+    @JsonProperty("sa_address_line_1")
+    private String saAddressLine1;
+
+    @JsonProperty("sa_address_line_2")
+    private String saAddressLine2;
+
+    @JsonProperty("sa_address_care_of")
+    private String saAddressCareOf;
+
+    @JsonProperty("sa_address_country")
+    private String saAddressCountry;
+
+    @JsonProperty("sa_address_locality")
+    private String saAddressLocality;
+
+    @JsonProperty("sa_address_po_box")
+    private String saAddressPoBox;
+
+    @JsonProperty("sa_address_postal_code")
+    private String saAddressPostalCode;
+
+    @JsonProperty("sa_address_premises")
+    private String saAddressPremises;
+
+    @JsonProperty("sa_address_region")
+    private String saAddressRegion;
+
+    @JsonProperty("identification_country_registration")
+    private String identificationCountryRegistration;
+
+    @JsonProperty("identification_legal_authority")
+    private String identificationLegalAuthority;
+
+    @JsonProperty("identification_legal_form")
+    private String identificationLegalForm;
+
+    @JsonProperty("identification_place_registered")
+    private String identificationPlaceRegistered;
+
+    @JsonProperty("identification_registration_number")
+    private String identificationRegistrationNumber;
+
+    public String getType() {
+        return type;
+    }
+
+    public void setType(String type) {
+        this.type = type;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getDateBecameInterestedPersonDay() {
+        return dateBecameInterestedPersonDay;
+    }
+
+    public void setDateBecameInterestedPersonDay(String dateBecameInterestedPersonDay) {
+        this.dateBecameInterestedPersonDay = dateBecameInterestedPersonDay;
+    }
+
+    public String getDateBecameInterestedPersonMonth() {
+        return dateBecameInterestedPersonMonth;
+    }
+
+    public void setDateBecameInterestedPersonMonth(String dateBecameInterestedPersonMonth) {
+        this.dateBecameInterestedPersonMonth = dateBecameInterestedPersonMonth;
+    }
+
+    public String getDateBecameInterestedPersonYear() {
+        return dateBecameInterestedPersonYear;
+    }
+
+    public void setDateBecameInterestedPersonYear(String dateBecameInterestedPersonYear) {
+        this.dateBecameInterestedPersonYear = dateBecameInterestedPersonYear;
+    }
+
+    public String getDateBecameInterestedPerson() {
+        if (type.equals("Corporate Interested Person")
```

Comment:
rather than have the literal string "Corporate Interested Person" in the Dto, I would either make the strings in BeneficialOwnerType enum into public constants and reference the string from there or in the same enum, add getter for the private field and use it like CORPORATE_INTERESTED_PERSON.getBeneficialOwnerType()